### PR TITLE
Use languages for Navbar Color

### LINKED ISSUE #2692: Navbar Colour are hard coded
### Describe the issue

Navbar Colour text are hard coded with no translations

### NamelessMC Version

Development version

### To Reproduce

_No response_

### Expected Behaviour

_No response_

### Screenshots

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by PadowYT2**
Closes #2692

### CODE REVIEW

**Comment by partydragen** [comment]
I myself liked the array better ¯\_(ツ)_/¯ and language colors to be colors in general as its just plain color text so it can be used anywhere else too and not just navbar color

like "general\/red": "Red",

**Comment by PadowYT2** [comment]
> I myself liked the array better ¯_(ツ)_/¯ and language colors to be colors in general as its just plain color text so it can be used anywhere else too and not just navbar color
> 
> like "general/red": "Red",

So should I make from `admin/` to `general/`?

**Comment by partydragen** [comment]
> > I myself liked the array better ¯_(ツ)_/¯ and language colors to be colors in general as its just plain color text so it can be used anywhere else too and not just navbar color
> > like "general/red": "Red",
> 
> So should I make from `admin\/` to `general\/`?

Well the term name too to just be the actual color name instead navbar_color_....

**Comment by PadowYT2** [comment]
> > > I myself liked the array better ¯_(ツ)_/¯ and language colors to be colors in general as its just plain color text so it can be used anywhere else too and not just navbar color
> > > like "general/red": "Red",
> > 
> > 
> > So should I make from `admin\/` to `general\/`?
> 
> Well the term name too to just be the actual color name instead navbar_color_....

Like 402399a

**Comment by samerton** [comment]
I agree that it's much easier to manage with an array (for example if we want to add a new colour)

Could we keep the array and add the language terms into that instead?

**Comment by PadowYT2** [comment]
> 

I've tried doing that, but {$} doesn't work in arrays

File: custom/templates/DefaultRevamp/template_settings/settings.php [hunk 1]
```diff
@@ -44,26 +44,81 @@
     $cache->store('navbarColour', $navbarColour);
 }
 
+$nav_colours = [
+    0 => [
```
[the remark this answers is not in the document]
Comment (PadowYT2) [reply custom/templates/DefaultRevamp/template_settings/settings.php:48]: Maybe an example? by removing `x => [` will have errors

[the remark this answers is not in the document]
Comment (Derkades) [reply custom/templates/DefaultRevamp/template_settings/settings.php:48]: Change
```php
[
  0 => 'something',
  1=> 'something',
]
```
to
```php
[
  'something', 
  'something',
]
```

**Review by samerton — APPROVED** [review]

**Review by Aberdeener — APPROVED** [review]
Could be cleaned up a little more since each `$nav_colours` entry is the exact same except for the colour name, but we can do this post-merge.
Thanks!

**Pull request merged**